app/services/electoral_service.rb

```diff
@@ -46,17 +46,17 @@ def assemble_error(exception)
     end
   end
 
-  def report_error(code)
-    GovukStatsd.increment("elections.errors.#{code}")
+  def report_status_code(code)
+    GovukStatsd.increment("elections_api.#{monitoring_path}.#{code}")
```

Comment:
Is it fine that before you changes we were logging `elections` and `elections_api` but now we're just logging `elections_api` ?

Reply:
Yep, I tweaked them so that they line up. It's arbitrary what we call them though.